test/conformance/ingress/util.go

```diff
@@ -1065,7 +1074,7 @@ func CreateDialContext(ctx context.Context, t *testing.T, ing *v1alpha1.Ingress,
 		t.Fatalf("Unable to retrieve Kubernetes service %s/%s: %v", namespace, name, err)
 	}
 
-	dial := network.NewBackoffDialer(dialBackoff)
+	//dial := network.NewBackoffDialer(dialBackoff)
```

Comment:
commited my accident?

Reply:
this was also added by mistake as part of my local testing, took it out